services/IBF-pipeline/pipeline/lib/pipeline/rainfalldata.py

```diff
@@ -261,21 +259,21 @@ def findTrigger(self):
                                 how='left', op='intersects')
 
         # interpolate NaN cells
-        known = ~np.isnan(compare_gdf['forecast_time'])
+        known = ~np.isnan(compare_gdf['leadTime'])
         unknown = ~known
         z = griddata((compare_gdf['longitude'][known], compare_gdf['latitude'][known]),
-                     compare_gdf[self.TRIGGER_RP_COLNAME][known],
+                     compare_gdf['triggerLevel'][known],
                      (compare_gdf['longitude'][unknown], compare_gdf['latitude'][unknown]))
-        compare_gdf[self.TRIGGER_RP_COLNAME][unknown] = z.tolist()
+        compare_gdf['triggerLevel'][unknown] = z.tolist()
 
         compare_gdf[str(str(self.leadTimeLabel) + '_pred')] = np.where(
-            (compare_gdf['mean_by_day'] > compare_gdf[self.TRIGGER_RP_COLNAME]), 1, 0)
+            (compare_gdf['mean_by_day'] > compare_gdf['triggerLevel']), 1, 0)
         compare_gdf['fc_day'] = compare_gdf['fc_day'].astype(str)
         df_trigger = compare_gdf.filter(
             ['latitude', 'longitude', 'geometry', str(str(self.leadTimeLabel) + '_pred')])
 
         # out = df_trigger.to_json()
-        # output_name = '%s_%sday_'%(runcycle_day, self.leadTimeLabel) +self.TRIGGER_RP_COLNAME
+        # output_name = '%s_%sday_'%(runcycle_day, self.leadTimeLabel) + 'triggerLevel'
```

Comment:
@jannisvisser Commented code can be removed. If needed in the future then we look for these in the git history.

Reply:
done (when committed). I agree with your point of course. In this particular case though, I was adhering already a little bit to the idea that Phuoc is the owner of this code, and I changed only things that were needed to make things work. Anyway, it's done.